runtime/gc_glue_java/HeadlessMixedObjectScanner.hpp

```diff
@@ -115,21 +135,18 @@ class GC_HeadlessMixedObjectScanner : public GC_ObjectScanner
 	virtual fomrobject_t *
 	getNextSlotMap(uintptr_t *slotMap, bool *hasNextSlotMap)
```

Comment:
This whole method could really be under #else of #if LEAF

Reply:
I see that it's pure virtual in base Scanner class, so this won't be possible before doing the same #else trick over there.